bin/init.js

```diff
@@ -3,5 +3,5 @@ const defaults = {id: '1', moderation: 'pre'};
 
 Setting.update({id: '1'}, {$setOnInsert: defaults}, {upsert: true}).then(() => {
   console.log('created settings object.');
-  process.exit();
+//  process.exit(); // Removed to satisfy hooks
```

Comment:
This should be resolved upstream, remove comment and merge with master to resolve.